src/main/java/AccountEditor.java

```diff
@@ -0,0 +1,17 @@
+import java.util.HashMap;
+import java.util.Objects;
+
+public class AccountEditor {
+
+    public void changePassword(Person user, String newPass) {
+        user.setUserPassword(newPass);
+    }
+
+    public boolean securityCheck(Person user, String answer) {
+        return Objects.equals(user.getTrustedAnswer(), answer);
+    }
+
+    public boolean userExists(String name, HashMap<String, Person> data) {
```

Comment:
i think we have this same method already in AccountCreator, although i can see its usefulness here. would it make sense to merge the two classes? uncertain

Reply:
I have opted to keep the two classes separate for now.